packages/orbit-components/src/primitives/ButtonPrimitive/common/getIconContainer.ts

```diff
@@ -45,9 +37,19 @@ const getIconContainer: getIconContainerType = ({
   size = SIZE_OPTIONS.NORMAL,
   iconForeground,
 }) => {
-  const onlyIcon = Boolean(iconLeft && !children);
-  const sizeIcon = size === ICON_SIZES.SMALL ? ICON_SIZES.SMALL : ICON_SIZES.MEDIUM;
+  let sizeIcon: ICON_SIZES;
```

Comment:
we usually try to avoid using let variables and switch/case, we prefer to have it declarative way, instead of imperative. Rewrite it to a function that will return the values you need 🙏🏻